share/CWW_E_ext_more.sed

```diff
@@ -0,0 +1,278 @@
+#  E SED of Coleman, Wu, and Weedman (1980)
```

Comment:
This file looks good.  I think you still need to update the old file (CWW_E_ext.sed) in this directory too though, since the thinning algorithm recently changed.  In particular, I noticed that the "less" thinned version starts at 137 Ang, and the more thinned version starts at 91 Ang, which is a bit counter-intuitive.